libs/shared/contentful/src/__generated__/fragment-masking.ts

```diff
@@ -1,12 +1,14 @@
-import { ResultOf, DocumentTypeDecoration, TypedDocumentNode } from '@graphql-typed-document-node/core';
+import {
```

Comment:
For the reviewer: I already had @julianpoy double check me that these auto-generated files looked alright. Not great that prettier diffs made it here but this is what happened after a `yarn install` for me.